src/tests/Dashboard/action/dictionaryAction.test.js

```diff
@@ -381,6 +382,49 @@ describe('Test suite for dictionary actions', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+
+  it('should retire a concept when the retire button is clicked', () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: sampleRetiredConcept,
+      });
+    });
+    const store = mockStore({ payload: {} });
```

Comment:
Where are is the simulation of clicking the retire button?

Reply:
`DictionaryConcepts.test.jsx` lines 528 - 621

Reply:
I mean in this very test?

Reply:
This is the Action that is triggered when the button is clicked. The test is only for the Action and the button click test is in the lines I mentioned. Therefore the simulation for the button click is not (and should not be) in this test.

Reply:
Then i find this confusing "should retire a concept when the retire button is clicked"

Reply:
I understand the concern. Should I change it to `"should retire a concept when the retireConcept method is triggered"`?

Reply:
I have changed the title to `should retire a concept when the retireConcept action is triggered with the true argument`